doc/man7/OSSL_PROVIDER-legacy.pod

```diff
@@ -15,14 +15,10 @@ We can consider this the retirement home of cryptographic algorithms.
 
 =head2 Properties
 
-The implementations in this provider specifically have these property
+The implementations in this provider specifically have this property
 defined:
 
-=over 4
-
-"provider=legacy"
-
-=back
+    provider=legacy
```

Comment:
... and here

Reply:
sure, but I hope he changes "these" to "this" :)

Reply:
I, er, haven't noticed.  Feel free to hop in and look :wink: